src/views/memory-webview-main.ts

```diff
@@ -101,19 +92,18 @@ export class MemoryWebview {
             'memory.js'
         ));
 
-        // webview.cspSource does not include all CSP sources for VS Code Web
-        const webviewUri = panel.webview.asWebviewUri(this.extensionUri);
-        const baseSource = `${webviewUri.scheme}://${webviewUri.authority}`;
-        const cspSrc = `${panel.webview.cspSource} ${baseSource}`;
```

Comment:
@thegecko, in my testing (VSCode 1.71), this caused an error log in the dev tools:
![image](https://user-images.githubusercontent.com/62660806/216679322-50911dae-7bd4-466a-a612-f3e9916f96a2.png)

It looks like the Webview's `cspSource` should cover what's constructed there:
`https://file+.vscode-resource.vscode-cdn.net`
should be covered under `https://*.vscode-cdn.net`
Unless your goal is to also add the `file` scheme?

Reply:
I assume you tested this on the desktop only? There are situations in the web version where cspSource doesn't include the sources from the marketplace correctly. Annoyingly I seem to remember this is only when running in production, too.

Reply:
This works fine in development using a browser, so happy to revisit this in future if we have errors loading assets from the published extension in the marketplace